hack/configapi/cmd/root.go

```diff
@@ -56,6 +56,7 @@ func newRootCmd() *cobra.Command {
 	}
 	rootCmd.Flags().StringVarP(&versionFilePath, "version-file", "f", "", "File path to the version json file.")
 	rootCmd.Flags().BoolVar(&force, "force", false, "force to upload version regardless of comparison to latest API value.")
+	rootCmd.Flags().StringP("upload-date", "d", "", "force to upload version regardless of comparison to latest API value.")
```

Comment:
the description does not match the flag. also: I feel like the flag should imply `force` being set.